RELEASE_NOTES.md

```diff
@@ -14,6 +14,8 @@
 
 ### New hardware support
 
+* CSM 1.6.0 does not support servers with NVIDIA CPUs and GPUs. Systems with these servers should not be upgraded to CSM 1.6.0.
+
```

Comment:
It would be best to omit this from the "New hardware support" section since it is false-positive; it's counter-intuitive to see "unsupported" items listed in "new hardware support."
```suggestion
```

Reply:
Thanks, I made this change, on a new commit.

Reply:
Where should it go? Should I make a new section? Maybe Unsupported Hardware section?

Reply:
Updated text to be a link (to known issues) as suggested in a different comment.